<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TradeDialog</class>
 <widget class="QDialog" name="TradeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid; 
border-color: rgb(1, 25, 147);  
color: rgb(1, 25, 147);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input Token</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="input_token_comboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_outputToken">
         <property name="text">
          <string>Output Token</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="output_token_comboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_protocol">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="protocol_comboBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_pool">
       <property name="text">
        <string>Pool</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pool_comboBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_quantity_lineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(169, 169, 169);</string>
     </property>
     <property name="placeholderText">
      <string>Enter input quantity here</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_outputQuantity">
       <property name="text">
        <string>Output quantity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="output_quantity_lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_slippage">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_slippage">
       <property name="text">
        <string>Slippage (%): </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_slippage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Trade</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
